demisto_sdk/commands/generate_docs/generate_readme_template.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from demisto_sdk.commands.generate_docs.readme_templates import (
+    HTTP_COLLECTOR,
+    SYSLOG,
+    XDRC,
+)
+
+logger = logging.getLogger("demisto-sdk")
+
+
+def generate_readme_template(input_path: str, readme_template: str):
+    file_object = open(input_path, "a")
```

Comment:
please use `with open(input_path, "a") as file_object` so that we don't have to remember to close it.